api/models/User.js

```diff
@@ -31,4 +40,100 @@ userSchema.statics.create = function(name) {
 	});
 };
 
+userSchema.statics.read = function(GoogleID)
+{
+	return new Promise((resolve , reject) => {
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}, (err,user)=>{
+			if(err) reject(err);
+			else resolve(user);
+		});
+	});
+}
+
+userSchema.statics.delete = function(GoogleID)
+{
+	return new Promise((resolve, reject)=>{
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}).remove((err,offer)=>{
```

Comment:
What is this parameter after removing `offer`? Should we call resolve in the promise otherwise the method won't return?